pontoon/sync/formats/xliff.py

```diff
@@ -89,10 +89,26 @@ def save(self, locale):
         for entity in self.entities:
             entity.sync_changes()
 
+        locale_code = locale.code
+
+        # TODO: should be made iOS specific
```

Comment:
At some point in the future we'll need to externalize these kinds of hacks. We have another one in silme.py, used in Firefox for Android. Something in the style of https://bugzilla.mozilla.org/show_bug.cgi?id=1209375.